sa/db/boulder_sa/20230419000000_CombinedSchema.sql

```diff
@@ -81,9 +111,10 @@ CREATE TABLE `issuedNames` (
   `serial` varchar(255) NOT NULL,
   `renewal` tinyint(1) NOT NULL DEFAULT 0,
   PRIMARY KEY (`id`),
-  KEY `reversedName_notBefore_Idx` (`reversedName`,`notBefore`),
-  KEY `reversedName_renewal_notBefore_Idx` (`reversedName`,`renewal`,`notBefore`)
```

Comment:
SRE still have the `reversedName_renewal_notBefore_Idx` index in staging and production.

Reply:
They only having it in Staging; Production only has the `reversedName_notBefore_Idx`.